templates/model-driven-operations/index.html

```diff
@@ -0,0 +1,473 @@
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Juju: The software that drives your software</h2>
+      <p>Juju is the toolkit for model-driven operations. 100% open source. Supports public cloud, private cloud, multi-cloud and all types of environments. Juju runs applications on bare-metal, virtual machines and containers with a refreshingly simple, consistent user experience.</p>
+      <a href="https://juju.is" class="p-button--positive">Learn moren about Juju</a>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width">
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>“On rails” critical business applications, services and infrastructure</h2>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Deploy and maintain complex software stacks without toil and brittle configurations thanks to powerful and easy-to-understand modelling capabilities.</li>
+        <li class="p-list__item is-ticked">Reduce errors by composing reusable operators with years of operational insight built into them.</li>
+        <li class="p-list__item is-ticked">Deploy faster and repeatably with tools designed for end-to-end automation.</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width">
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>The model-driven operator manifesto</h2>
+      <h3>Better operators through open community standards</h3>
+      <p>Our goal is to improve DevSecOps with open-source operators for widely used software. These shared values help ensure high-quality code and a healthy community.</p>
+      <a class="p-link--external" href="https://juju.is/model-driven-operations-manifesto">Read the Model-driven manifesto</a>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width">
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2 class="u-align--center u-sv4">Automate operations everywhere</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-4">
+      <div class="u-hide--small u-align--center u-sv2">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/04d70172-Multi-Cloud.svg",
+          alt="Multi-cloud",
+          width="263",
+          height="150",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h3>Public clouds and Multi-cloud</h3>
+      <p>Combine the advantages of public cloud services while enjoying long-term cost savings, increased flexibility and reduced vendor lock-in.</p>
+    </div>
+    <div class="col-4 ">
+      <div class="u-hide--small u-align--center u-sv2">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/9ed50294-Virtual+machine.svg",
+          alt="Virtual machine",
```

Comment:
Please change the `alt` to "Juju manages Virtual Machines running on various hypervisors"